src/components/Database/DatabaseDefaultRoute.tsx

```diff
@@ -16,21 +16,42 @@
 
 import React from 'react';
 import { connect } from 'react-redux';
-import { Redirect } from 'react-router-dom';
+import {
+  Switch,
+  Route,
+  useRouteMatch,
+  Redirect,
+  match,
+} from 'react-router-dom';
+
 import { AppState } from '../../store';
+import Database from './index';
 
 export interface PropsFromState {
   projectId?: string;
 }
 
 export type Props = PropsFromState;
 
+const DatabaseRoute: React.FC<{ match: match<{ namespace: string }> }> = ({
```

Comment:
nit: inline this.

Reply:
Inline it on :14?

Reply:
I've added a suggested edit below to clarify.